notebooks/TFSegFormer.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<p>If this notebook is supposed to run on Colab then I guess we need to show the pip installation from source (specifying the <code>tf-segformer</code> branch).  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I have included instructions in README of this PR. But I will try to run this on colab as well</p>